### PR TITLE
Added functionality to job postings save/unsave button

### PR DESCRIPTION
**Pull request by Edward-Boguslavsky**
When clicking the save/unsave button, the front end now talks to the database and stores/removes the saved posting information under the selected user.

Note: When the postings/saved tabs are clicked on the job postings page, the saved postings are not updated unless the whole page is refreshed first. Nikki and I are actively trying to fix this

### CODE REVIEW

**Review by nikkikapadia — CHANGES_REQUESTED** [review]
plz only post PRs when you have the full functionality; implement the list updating first and then I can review it.

**Comment by Edward-Boguslavsky** [comment]
@nikkikapadia Both tabs now update properly

File: launchpad-frontend/src/jobsPage/jobsPage.js [hunk 1]
```diff
@@ -327,17 +327,30 @@ function JobExpanded({ postingSelected, updateList, postings, saved }) {
   const expandedPosting = postings.find(
     (post) => post.postingId === postingSelected
   );
-  const [save, setSave] = React.useState(
+  /*const [save, setSave] = React.useState(
```
Comment (nikkikapadia) [inline launchpad-frontend/src/jobsPage/jobsPage.js:330]: take out commented code plz

File: launchpad-frontend/src/jobsPage/jobsPage.js [hunk 2]
```diff
@@ -327,17 +327,30 @@ function JobExpanded({ postingSelected, updateList, postings, saved }) {
...
     expandedPosting ? expandedPosting.saved : false
-  );
+  );*/
 
   const isSavedPosting = expandedPosting ? expandedPosting.saved : false;
 
   const handleSave = () => {
-    expandedPosting.saved = !save;
-    setSave(!save);
+    //expandedPosting.saved = !save;
```
Comment (nikkikapadia) [inline launchpad-frontend/src/jobsPage/jobsPage.js:337]: comments again

**Review by nikkikapadia — APPROVED** [review]
approved for functionality but take a look at the comments

**Pull request merged**